filament/src/details/MorphTargetBuffer.cpp

```diff
@@ -77,22 +77,22 @@ inline size_t getSize(size_t vertexCount) noexcept;
 
 template<>
 inline size_t getSize<VertexAttribute::POSITION>(size_t vertexCount) noexcept {
-    const size_t stride = getWidth(vertexCount) * sizeof(float3);
+    const size_t strideInPixels = getWidth(vertexCount);
```

Comment:
I'm okay with renaming to `strideInPixels` but this *is* the convention we use everywhere in filament. In the rare case that we need to deal with a stride that is in byte, we explicitly call it `strideInBytes`.
So to me, this is not necessary and different from other places in filament.